src/main/java/appeng/spatial/StorageChunkProvider.java

```diff
@@ -51,7 +51,7 @@ public Chunk generateChunk( final int x, final int z )
 		final Chunk chunk = new Chunk( this.world, x, z );
 
 		final byte[] biomes = chunk.getBiomeArray();
-		Biome biome = AppEng.instance().getRegistration().getStorageBiome();
+		Biome biome = AppEng.instance().getStorageBiome();
```

Comment:
This might be a good candidate for the API. Having a explicit source instead of trying to infer it somehow might be beneficial for certain mods.